libsql-server/tests/namespaces/meta.rs

```diff
@@ -0,0 +1 @@
+
```

Comment:
reserved for some tests that I will do as follow up

Reply:
I think this will create more confusion than value. I don't think anyone will still the meta.rs file so I would just drop this from the PR

Reply:
I've added a test here, these will be expanded upon in follow up PRs